embedding-calculator/Makefile

```diff
@@ -2,13 +2,14 @@ SHELL := /bin/bash
 .DEFAULT_GOAL := default
```

Comment:
It looks like in this file lots of temporary code, please revert it

Reply:
Made it the same as it is in 1.0.x